src/Umbraco.Core/Configuration/Models/NuCacheSettings.cs

```diff
@@ -2,6 +2,6 @@
 {
     public class NuCacheSettings 
     {
-        public string BTreeBlockSize { get; set; }
+        public int? BTreeBlockSize { get; set; }
```

Comment:
Does NuCache provide a default value? Does it already validate input and clamp it to a range? Is this something that should even be set?

Reply:
It does, yes: `4096`.  Not going to pretend I particularly understand it!  But yes, there's some validation of it already under `BTree.GetBlockSize()`.  We could consider migrating this into a configuration validator class too, but I've not done that as it's a little more involved/specific than your typical validation.

Reply:
It seems like the kind of configuration I would leave well alone as setting an arbitrary value without profiling could be harmful.

Reply:
Found a reference to it being made configurable here: https://github.com/umbraco/Umbraco-CMS/issues/4998

Reply:
Whoa!  That means there’s an issue requiring a fix.